size-auditor/size-auditor.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Size auditor Configuration",
+  "description": "Configuration file for the Size Auditor tool",
+  "type": "object",
+  "properties": {
+    "sizeTolerance": {
+      "description": "The tolerance limit above which Size Auditor would fail a pull request",
+      "type": "number"
+    },
+    "trackingEntity": {
+      "description": "The entity that is audited for size regressions by Size Auditor",
+      "type": "string",
+	  "pattern": "/a@(Asset|Chunk|Assembly)\b/"
+    },
+    "devopsDropFolderName": {
+      "description": "The name of the drop location containing build artifacts",
```

Comment:
Specify that it is a folder name where the artifact drop is located. It is NOT a relative or absolute path (currently).

Reply:
thanks! I can see how this could be misleading and lead someone to pass in a path.